engine/include/Engine/Core/Rendering/Window/WindowGLFW.hpp

```diff
@@ -53,7 +53,7 @@ class Window
      * @param height
      * @return
      */
-    void getSize(int& width, int& height) const noexcept;
+    void getSize(int width, int height) const noexcept;
```

Comment:
The new version can't return the size of the window without pointers, references, or a return value
```suggestion
    void getSize(int& width, int& height) const noexcept;
    // Or
    Math::Vec2 getSize() const noexcept;
```